microsoft/atlas-design
# fix lint with quotes and ampersands

Task: task-511585

Link: preview-277

Previous build with a lint error committed: https://github.com/microsoft/atlas-design/runs/4086544836?check_suite_focus=true

## Testing

See link above.

1. Pull locally.
2. Make a CSS change that will fail link (like putting height last in .button)
3. Run lint.
4. Should fail.

All DevOps here, so no changeset.
